docs/redocly.yaml

```diff
@@ -0,0 +1,25 @@
+theme:
+  openapi:
+    disableSearch: false
+    disableGoogleFont: "true"
```

Comment:
❓ does this result in sth different if you put the literal `true` instead of the string? if not, we should go with the literal.

Reply:
unfortunatly this is needed due to a bug in redocly